caproto/commandline/put.py

```diff
@@ -84,8 +86,10 @@ def main():
         else:
             set_handler(color=not args.no_color, level='DEBUG')
     logger = logging.LoggerAdapter(logging.getLogger('caproto.ch'), {'pv': args.pv_name})
-
-    if args.array:
+    if args.S:
```

Comment:
```suggestion
    if args.as_string:
```

To go with @klauer 's suggestion above.